bors.toml

```diff
@@ -0,0 +1,16 @@
+status = [
+  "ci/gitlab/%",
+  "continuous-integration/gitlab-build-and-test"
+]
+# The default is one hour.
+timeout_sec = 600
+
+# Do not allow pull requests with these labels to be merged.
+block_labels = [
+    "question"
+]
```

Comment:
not sure if we actually need that.

Reply:
It was just a harmless sample in case of you will want something like that. Can remove.